.werft/observability/monitoring-satellite.ts

```diff
@@ -114,7 +114,12 @@ export class MonitoringSatelliteInstaller {
                         {
                             "gitURL": "https://github.com/gitpod-io/observability",
                             "path": "monitoring-satellite/manifests/probers",
-                    }],
+                        },
+                        {
+                            "gitURL": "https://github.com/gitpod-io/gitpod",
+                            "path": "operations/observability/mixins/meta/rules",
+                        },
```

Comment:
These same lines will need to be added to all monitoring-satellite apps that we have in argocd